### PR TITLE
feat: add readness score

### PR DESCRIPTION
**Pull request by narenaryan**
This MR includes below changes:

1. Attach readability label next to prompt
2. Replace fetch with ajax.post to remove CSRF workaround

### CODE REVIEW

**Comment by narenaryan** [comment]
Screenshot 2023-04-11 at 8 36 21 AM
